ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -94,6 +94,12 @@
   Dispatches on the type. By default, returns service as is."
   ::t-service/type)
 
+(defmethod transform-save-by-type :parking [service]
+  (update-in service [::t-service/parking]
```

Comment:
Could you add an explanation why :maximum-stay-unit and -unlimited maps are removed?

Reply:
Good point. Forgot to remove old obsolete code.